index.html

```diff
@@ -13,139 +13,154 @@
 </head>
 
 <body>
-    <header class="container-header">
-        <a id="name" href=""><img class="mylogo" src="attributes/mylogo.png" alt="logo" /></a>
-        <nav><img class="union" src="attributes/union.png" alt="union_icon" /></nav>
-        <nav class="desktop_headicons">
-            <a class="header_btn" href="">Portfolio</a>
-            <a class="header_btn" href="#about_text">About</a>
-            <a class="header_btn" href="#full_name">Contact</a>
-            <a class="header_btn" href=""><img id="mail" src="attributes/mailicon.png" alt="mailicon" /></a>
+    <header>
+        <nav class="desktop-nav">
```

Comment:
[Optional]
- [ ] It's better if we can't scroll when we open the mobile menu 👍 

![Animation](https://user-images.githubusercontent.com/50721479/201730079-736e4d62-b781-4db8-b455-4fe26b320bba.gif)